packages/playground/src/components/layout/top-bar-widget.tsx

```diff
@@ -33,32 +33,44 @@ interface BreadcrumbItemProps {
   isLast: boolean
 }
 
-const ProjectDropdown: React.FC<{ isPrimary: boolean; projects: Project[] }> = ({ isPrimary, projects }) => (
-  <DropdownMenu>
-    <DropdownMenuTrigger className="group flex items-center gap-2 outline-none">
-      <Avatar className="w-7 h-7">
-        <AvatarFallback>{getInitials(projects[0].name, 1)}</AvatarFallback>
-      </Avatar>
-      <BreadcrumbLink
-        className={cn('font-medium', { 'text-primary': isPrimary, 'text-navbar-text-secondary': !isPrimary })}>
-        {projects[0].name}
-      </BreadcrumbLink>
-      <Icon
-        name="chevron-down"
-        size={10}
-        className={cn('chevron-down', {
-          'text-primary': isPrimary,
-          'text-navbar-text-secondary group-hover:text-primary': !isPrimary
-        })}
-      />
-    </DropdownMenuTrigger>
-    <DropdownMenuContent align="start" className="mt-1.5">
-      {projects.map(project => (
-        <DropdownMenuItem key={project.id}>{project.name}</DropdownMenuItem>
-      ))}
-    </DropdownMenuContent>
-  </DropdownMenu>
-)
+const ProjectDropdown: React.FC<{ isPrimary: boolean; projects: Project[] }> = ({ isPrimary, projects }) => {
+  const [selectedProject, setSelectedProject] = useState<string>(projects[0]?.name || '')
```

Comment:
default should not be the first project.. default should be "no project selected"